SF iOS/SF iOS/SwipableNavigationContainer.m

```diff
@@ -100,15 +96,21 @@ - (UIViewController *)pageViewController:(UIPageViewController *)pageViewControl
 }
 
 - (void)controller:(nonnull GroupCollectionViewController *)controller tappedGroup:(nonnull Group *)group {
-     [self.mainNav pushViewController:[self viewControllerForGroup:group]
-                            animated:true];
+	[self.mainNav presentViewController:[self viewControllerForGroup:group] animated:YES completion:nil];
```

Comment:
I think I missed the main thought here. We're now presenting rather than pushing because that's the only way to prevent the swipe behavior right? 

From a design and flow perspective I'm trying to think if this is what Apple would do. Is there precedence within the apps that are available within the App Store? 

On one hand it feels like events are a detail view of groups, and the event detail is yet a further detail of the event feed.

On the other I could see the selection of groups switching entire contexts so I'm kind of on board with that approach as well.